packages/contracts-bedrock/package.json

```diff
@@ -21,8 +21,7 @@
     "test": "pnpm build:differential && pnpm build:fuzz && forge test",
     "coverage": "pnpm build:differential && pnpm build:fuzz && forge coverage",
     "coverage:lcov": "pnpm build:differential && pnpm build:fuzz && forge coverage --report lcov",
-    "gas-snapshot:no-build": "forge snapshot --no-match-test 'testDiff|testFuzz|invariant|generateArtifact'",
-    "gas-snapshot": "pnpm build:differential && pnpm build:fuzz && pnpm gas-snapshot:no-build",
+    "gas-snapshot": "forge snapshot --match-contract GasBenchMark",
```

Comment:
Although all the tests that this picks up are currently stored in `Benchmark.t.sol`, this will catch any contract with the string `GasBenchMark` in it, enable benchmarks to be written in any test file.